spec/e2e/enum_spec.rb

```diff
@@ -70,6 +70,16 @@
       expect(ids).to include(file2.id)
     end
 
+    it 'supports arrays' do
```

Comment:
Uhm... I'm not sure I was clear...
When I implemented enums, I defined a test for array parameters, that is veeeery similar to this one. Since `where` with array parameters was not yet implemented, I left it with a `skip` message, so when someone else would implement it, he could use it.
I mean this:
https://github.com/neo4jrb/neo4j/blob/enum_array_values/spec/e2e/enum_spec.rb#L83